example/gardener-local/kind/cluster/templates/_extra_port_mappings.tpl

```diff
@@ -17,38 +17,22 @@
   hostPort: 9443
 {{- end }}
   listenAddress: {{ .Values.gardener.seed.istio.listenAddressDefault }}
-- containerPort: 30444
-{{- if or (eq .Values.environment "local") .Values.gardener.controlPlane.deployed }}
+{{- if .Values.gardener.seed.istio.additionalListenAddresses }}
+{{- range $i, $listenAddress := .Values.gardener.seed.istio.additionalListenAddresses }}
+- containerPort: {{ add 30444 $i }}
+{{- if or (eq $.Values.environment "local") $.Values.gardener.controlPlane.deployed }}
```

Comment:
Is there any reason why you added '$' in front of `.Values...`? The previous approach is still used in line 11.

Reply:
Yes, I doesn't work otherwise 😄 
Thanks for the hint, the duplication is further removed in 22ced3e7daf1f3a09ebe23357cfd295cb0483d40.